### PR TITLE
Feature/multiple assets

### PR DESCRIPTION
**Pull request by Kron1749**
# Task

Added option for user to add multiple assets at one time
Added option for pin assets

### CODE REVIEW

File: src/components/AssetList.vue [hunk 1]
```diff
@@ -19,6 +19,9 @@
         <asset-list-item
           :asset="item"
           :with-clickable-logo="withClickableLogo"
+          selectable="selectable"
```
Comment (Nikita-Polyakov) [inline src/components/AssetList.vue:22]: `:selectable="selectable"` to pass boolean prop

File: src/store/account/mutations.ts [hunk 2]
```diff
@@ -105,6 +105,14 @@ const mutations = defineMutations<AccountState>()({
   setAccountAssets(state, accountAssets: Array<AccountAsset>): void {
     state.accountAssets = accountAssets;
   },
+  setPinnedAsset(state, pinnedAccountAsset: AccountAsset): void {
+    state.pinnedAssets.push(pinnedAccountAsset);
+    storage.set('pinnedAssets', JSON.stringify(state.pinnedAssets));
```
Comment (Nikita-Polyakov) [inline src/store/account/mutations.ts:110]: Do we need to store full assets data? Maybe just an array of addresses

File: src/components/AddAsset/AddAssetDetailsCard.vue [hunk 3]
```diff
@@ -120,5 +156,20 @@ export default class AddAssetDetailsCard extends Mixins(TranslationMixin, Loadin
...
+    }
+    & > div:first-child {
+      margin-top: 14px;
+    }
+    & > div:last-child {
+      margin-bottom: 14px;
+    }
+  }
+  .asset-list-scrollbar {
+    height: 400px;
```
Comment (Nikita-Polyakov) [inline src/components/AddAsset/AddAssetDetailsCard.vue:172]: `calc(var(--s-asset-item-height--fiat)* 2 + 32px);`
To save some space & show 2 assets + part of the third

**Comment by smartdev914** [comment]
Hi @Nikita-Polyakov ,
I'm Corey, a former developer at Thorswap (https://app.thorswap.finance/), which is providing multichain transactions, including token swaps, liquidity pools, staking, borrowing, and yield farming across networks like Bitcoin, Ethereum, Solana, Avalanche, Arbitrum, Polkadot, Kujira, Dogecoin, Cosmos, BSC, Base, and more.

I am very interested in this polkaswap and would love to explore any opportunities to contribute to your team!

**Review by Nikita-Polyakov — APPROVED** [review]

**Pull request merged**